lib/fathead/jquery/output.txt

```diff
@@ -1,1919 +1,1919 @@
-.add()	A			jQuery Traversing\njQuery Miscellaneous Traversal							<section class="prog__container"><p>Create a new jQuery object with elements added to the set of matched elements.</p><pre><code>$(selector).add(selector) → jQuery\n$(selector).add(elements) → jQuery\n$(selector).add(html) → jQuery\n$(selector).add(selection) → jQuery\n$(selector).add(selector, context) → jQuery</code></pre></section>	https://api.jquery.com/add/
-.addBack()	A			jQuery Traversing\njQuery Miscellaneous Traversal							<section class="prog__container"><p>Add the previous set of elements on the stack to the current set, optionally filtered by a selector.</p><pre><code>$(selector).addBack(selector) → jQuery</code></pre></section>	https://api.jquery.com/addBack/
+	R	jQuery()										
```

Comment:
@thinker3197 can you ensure we check that the title has a length before creating an entry for it?